src/components/events/AccessibilityRoutingProtocolWidget.tsx

```
@@ -0,0 +1,265 @@
+
+export const MOCK_PENDING_REQUEST: AccessibilityRequestPayload = {
+  id: "req-asl-9402",
+  eventId: "evt-keynote-1",
+  eventTitle: "Annual Campus AI & Robotics Symposium 2026",
+  accommodationType: "asl_interpreter",
+  status: "routed_to_disability_services",
+  disabilityServicesTicketId: "DS-9402",
+  createdAt: new Date().toISOString(),
+};
+
+export const AccessibilityRoutingProtocolWidget: React.FC<AccessibilityRoutingProtocolWidgetProps> = ({
```

ESLint

[error] Replace `AccessibilityRoutingProtocolWidgetProps` with `⏎··AccessibilityRoutingProtocolWidgetProps⏎`

(prettier/prettier)